extensions/google-chrome/src/search-all.tsx

```diff
@@ -71,7 +71,7 @@ export default function Command() {
         {bookmarkData.length === 0 ? (
           <List.Item title="No bookmarks found" key={"empty bookmark list item"} />
         ) : (
-          tabData.map((tab) => (
+          bookmarkData.map((tab) => (
             <ChromeListItems.TabList key={tab.key()} tab={tab} useOriginalFavicon={useOriginalFavicon} />
```

Comment:
Switching to bookmarkData is correct. Bookmark data relies on a different component though, hence typescript complaining about the "key" type on the TabList.  Both the bookmarkData and `ChromeListItems.TabList` need switching. See [how the bookmark search component does it](https://github.com/raycast/extensions/blob/8533f11972392b6d22f69f073fdb2af6d8ffee10/extensions/google-chrome/src/search-bookmarks.tsx#L26C10-L26C36).

Reply:
PR https://github.com/raycast/extensions/pull/15332